apps/core/app/(default)/layout.tsx

```diff
@@ -11,6 +12,9 @@ export default function DefaultLayout({ children }: PropsWithChildren) {
       <main className="flex-1 px-6 2xl:container sm:px-10 lg:px-12 2xl:mx-auto 2xl:px-0">
         {children}
       </main>
+      <Suspense fallback={null}>
+        <ProductSheet />
+      </Suspense>
```

Comment:
Wrapping in suspense because of this: https://nextjs.org/docs/app/api-reference/functions/use-search-params#static-rendering